napari/layers/base/base.py

```diff
@@ -1594,9 +1595,51 @@ def _update_draw(
             corners[:, displayed_axes] = data_bbox_clipped
             self.corner_pixels = corners
 
-    def get_status(
+    def _get_source_info(self):
+        components = {}
+        if self.source.reader_plugin:
+            try:
+                components['layer_base'] = os.path.basename(self.source.path)
+            except KeyError:
+                components['layer_base'] = ''
+
+            components['source_type'] = 'plugin'
+            components['plugin'] = self.source.reader_plugin
```

Comment:
I would use display name of the plugin here so 

```python
from npe2 import plugin_manager as pm
pm.get_manifest(self.source.reader_plugin).display_name
```

Reply:
you might wanna `try/except` for now until we make the npe1 shim the only option